<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>321</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>311</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1d4e89;
color: White;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #1d4e89;
color: White;
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Leelawadee</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background : #1d4e89;
color: white;
border-radius:8px;</string>
   </property>
   <property name="text">
    <string>IT Core Checklist</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ccb logo.qrc"/>
 </resources>
 <connections/>
</ui>
